src/Common/Thermodynamics/states.jl

```diff
@@ -296,9 +301,11 @@ function PhaseEquil_ρθq(
     ρ::FT,
     θ_liq_ice::FT,
     q_tot::FT,
-    maxiter::Int = 36,
-    temperature_tol::FT = FT(1e-1),
-) where {FT <: Real}
+    maxiter::IT = nothing,
+    temperature_tol::FTT = nothing,
+) where {FT <: Real, IT <: ITERTYPE, FTT <: TOLTYPE(FT)}
+    maxiter === nothing && (maxiter = 36)
```

Comment:
Should we bump 36 to 40? Just to have the minimum amount of numbers (PhaseEquil_peq uses 40).

Reply:
We can, I forget exactly why the number is so large considering this is now used at runtime with the Anelastic1D model. I imagine that the ref profiles were not converging without such a large number.

Reply:
Yes, I know for a fact the `_peq` constructor is very slow and for some of the tests cases it fails to converge.I looked into this with @trontrytel but we could not find a reason. I think a large number for this constructor is fine since we use it only in the anelastic setup, and that one is not operational.

Reply:
That said, this applies to `_peq`, I don't know what's the situation with `_ρθq`, which I think is used for the Compressible, theta model.